src/app/resume/PDFViewer.tsx

```
@@ -0,0 +1,153 @@
+    };
+  }, [pdfFile]);
+
+  function onDocumentLoadSuccess({ numPages }: { numPages: number }): void {
+    setNumPages(numPages);
+  }
+
+  const handleFileChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const file = event.target.files?.[0];
+    if (file && file.type === 'application/pdf') {
+      // Revoke any previous object URL to avoid memory leaks
+      if (pdfFile && pdfFile.startsWith('blob:')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)